genie-client/src/test/java/com/netflix/genie/client/JobClientIntegrationTests.java

```diff
@@ -151,6 +155,70 @@ public void canSubmitJob() throws Exception {
     }
 
     /**
+     * Method to test submitting/killing a job.
+     *
+     * @throws Exception If there is any problem.
+     */
+    @Test
+    public void submitAndKillJob() throws Exception {
```

Comment:
Don't we have a test for killing the job in the JobRestControllerIntegrationTests?